packages/f-checkout/test/specs/component/f-checkout-collection.component.spec.js

```diff
@@ -0,0 +1,35 @@
+import forEach from 'mocha-each';
+import DemoControls from '../../../test-utils/component-objects/demo-controls';
+import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.component';
+
+describe('f-checkout "collection" component tests', () => {
+    forEach(['mobileNumber'])
+    .it.only('each fields error message should be displayed', field => {
```

Comment:
Just checking that the `.only` here isn't preventing any of the other tests from being run?